### PR TITLE
Remove wrong way of import DSG code

### PR DESCRIPTION
**Pull request by tupe12334**
Close: #4974 

## PR Details

This PR removes the old and wrong way to import DSG create-log code
## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -26,9 +26,6 @@
       "@amplication/data-service-generator": [
         "libs/data-service-generator/src/index.ts"
       ],
-      "@amplication/data-service-generator/*": [
```
Comment (abrl91) [inline tsconfig.base.json:29]: why did you remove it?

File: packages/amplication-data-service-generator-runner/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,6 @@
...
-WORKDIR /workspace/packages/amplication-code-gen-types
-COPY ./dist/packages/amplication-code-gen-types/package.json .
-RUN npm i --omit-dev
-COPY ./dist/packages/amplication-code-gen-types .
-
-WORKDIR /workspace/libs/data-service-generator
-COPY ./dist/libs/data-service-generator/package.json .
-RUN npm i --omit-dev
-COPY ./dist/libs/data-service-generator .
-
```
Comment (abrl91) [inline packages/amplication-data-service-generator-runner/Dockerfile:13]: why did you remove it?

File: packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { PluginInstallation } from "@amplication/code-gen-types";
-import { createLog } from "@amplication/data-service-generator/create-log";
+import { createLog } from "@amplication/data-service-generator";
```
Comment (abrl91) [inline packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts:2]: How the other changes helped you with this import?

File: packages/amplication-data-service-generator-runner/Dockerfile (hunk 2 above)
Comment (tupe12334) [reply packages/amplication-data-service-generator-runner/Dockerfile:13]: This is not required any more because we installing the DSG as a npm package

File: packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts (hunk 3 above)
Comment (tupe12334) [reply packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts:2]: Its blocking us to make the same mistake again

File: tsconfig.base.json (hunk 1 above)
Comment (tupe12334) [reply tsconfig.base.json:29]: This change blocks us from making the same mistake again

File: packages/amplication-data-service-generator-runner/Dockerfile (hunk 2 above)
Comment (abrl91) [reply packages/amplication-data-service-generator-runner/Dockerfile:13]: and what about the code-gen?

File: packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts (hunk 3 above)
Comment (abrl91) [reply packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts:2]: I mean, before the changes you have applied, you wouldn't be able to import `createLog ` from `@amplication/data-service-generator`. How these changes (in the other files) helped you to import it like that?

File: tsconfig.base.json (hunk 1 above)
Comment (abrl91) [reply tsconfig.base.json:29]: Why don't we need data-service-generator in the ts config?

Comment (tupe12334) [reply tsconfig.base.json:29]: We need it, but the main entry only, the index.ts

File: packages/amplication-data-service-generator-runner/Dockerfile (hunk 2 above)
Comment (tupe12334) [reply packages/amplication-data-service-generator-runner/Dockerfile:13]: Its come as a dependency of the DSG

File: packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts (hunk 3 above)
Comment (tupe12334) [reply packages/amplication-data-service-generator-runner/src/dynamic-package-installation.ts:2]: https://github.com/amplication/amplication/pull/4978/files#diff-6f1199c262f20ad5f5baec5d4f927a8c6f8162389220df492b17fac5945dce68

**Review by abrl91 — APPROVED** [review]

File: packages/amplication-data-service-generator-runner/project.json [hunk 4]
```diff
@@ -58,8 +58,9 @@
       },
       "dependsOn": ["generate-example-input-json"]
     },
-    "docker:build": {
+    "docker-build": {
```
Comment (overbit) [inline packages/amplication-data-service-generator-runner/project.json:61]: leave docker:build since it will benefit from the default dependencies like `pre build`, `build` and `postbuild` @tupe12334 @abrl91

File: packages/amplication-data-service-generator-runner/project.json [hunk 5]
```diff
@@ -58,8 +58,9 @@
...
       "executor": "@nx-tools/nx-docker:build",
+      "dependsOn": ["build"],
```
Comment (overbit) [inline packages/amplication-data-service-generator-runner/project.json:63]: not needed since it is defined as default in nx.json @tupe12334 @abrl91

File: packages/amplication-data-service-generator-runner/project.json (hunk 4 above)
Comment (abrl91) [reply packages/amplication-data-service-generator-runner/project.json:61]: this already was fixed
